lib/resty/mlcache.lua

```diff
@@ -821,7 +821,7 @@ function _M:get(key, opts, cb, ...)
         error("key must be a string", 2)
     end
 
-    if type(cb) ~= "function" and (not opts or not opts.cached_only) then
+    if cb and type(cb) ~= "function" then
```

Comment:
Let's be a lot stricter than this though, and only explicitly allow for either `function` or `nil`, nothing else, what do you say? I prefer not to leave any room for error in this library... (Including test cases differentiating `nil` vs `false`)

Reply:
👍